### PR TITLE
Load SDG regions from geo_regions.json.

### PR DESCRIPTION
**Pull request by keyurva**

### CODE REVIEW

File: server/webdriver/tests/standalone/sdg_sanity.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ class Mode:
 OUTPUT_DIR = "output"
 
 COUNTRIES_JSON_FILE = "https://code.officialstatistics.org/undata2/data-commons/frontend-tester/-/raw/main/UNSD-Website-skeleton/UNSD.Website/ClientApp/src/templates/DataCommons/config/countries.json?ref_type=heads&inline=false"
-REGIONS_JSON_FILE = "sdg_regions.json"
+REGIONS_JSON_FILE = "https://code.officialstatistics.org/undata2/data-commons/frontend-tester/-/raw/main/UNSD-Website-skeleton/UNSD.Website/ClientApp/src/templates/DataCommons/config/geo_regions.json?ref_type=heads&inline=false"
```
Comment (pradh) [inline server/webdriver/tests/standalone/sdg_sanity.py:70]: Also DC entities of type:  "UNGeoRegion", "ContinentalUnion", "Continent" ?  Can make a static list if you like.

**Review by pradh — APPROVED** [review]
Thanks.

**Review by dwnoble — APPROVED** [review]

File: server/webdriver/tests/standalone/sdg_sanity.py (hunk 1 above)
Comment (keyurva) [reply server/webdriver/tests/standalone/sdg_sanity.py:70]: We had made a list earlier - does this one work? https://github.com/datacommonsorg/website/blob/1a941e6d1b81957003cbda6d3a6adf6d4cbf4e8e/server/webdriver/tests/standalone/sdg_regions.json

Comment (pradh) [reply server/webdriver/tests/standalone/sdg_sanity.py]: Yes, if you've confirmed the content of `sdg_regions.json` is `geo_regions.json` + entities of those types?

**Pull request merged**